Project-3/Project#3_tf

```diff
@@ -0,0 +1,292 @@
+
+Ordinarily, that would seem fine. We could put aside whether Tuesday and Wednesday are references to days of the week, but even in ordinary uses of languages Tuesday and Wednesday could be proper names. (Think, e.g., Wednesday Addams. Maybe she has a cousin named Tuesday...?) Tuesday and Wednesday could also be the names of Sally's dogs. Under an open-world assumption, none of this possibilities could be ruled out right away. The dog names possibility, however, doesn't quite work without the preposition 'on.' So, we are back to Sally having an arm on one day, but not on the other. (Notice, interestingly, that the preposition 'on' could be in this expression even if we are referring to days of the week, but isn't.) Back to the logical ambiguity at play in the expression, one may note that Sally could be a person that only has one arm; the cause doesn't much matter. It would be true that Sally is an one-arm person in both instances asserted in the expression: Sally would have an arm on Tuesday, because Sally doesn't have two arms; and, by the same logic, it would be true that Sally does not have an arm on Wednesday if she only has one arm (and doesn't have two arms), because the expression is referring to the arm that Sally is missing on Wednesday, and referring to the arm that she is not missing on Tuesday.
+
+Even with ordinary language use, there is still another ambiguity inherent in the expression. Namely, how one goes about defining 'has.' There are at least three relevant layers of meaning possible here. Sally the arms-dealer *has* arms that she sells for profit. Sally the person who is missing an arm *has* one arm that came attached with her phsyical body. This same Sally might have a prosthetic arm that she puts on and takes off at will, so perhaps the expression could be interpreted to be referring to her taking on the prosthetic arm from one day to the next. (She could also be missing both arms, and the expression refers to the one prosthetic arm she got from her insurance.) She would still own the prosthetic arm, but not 'have' it with her or on her, when she takes the arm off, on Wednesdays, for whatever reason. Sally could also be a mannequin at the local retail store... (again, one could write a novel with so much ambiguity...)
+ 
+So, to modeling the expression:
+
+Sally is instance_of object (person, robot, mannequin...)
+Tuesday is instance_of one-dimensional temporal region 
+Wednesday is an instance of one-dimensional temporal region 
+Arm is an instance of fiat object part or an instance_of object (applies equally as well to person, robot, mannequin...)
+“Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
```

Comment:
Ah but this isn't an instance of occurrent. You must tease out the details of participation, having, etc. then talk about those things as related to the process in which Sally participates.